features/step_definitions/networking.rb

```diff
@@ -1,3 +1,5 @@
+require 'json'
```

Comment:
@rbbratta don't think you need to import `json`  you can use the `yaml` lib instead which is a superset of JSON